backend/Controllers/SightingsController.cs

```diff
@@ -30,4 +32,19 @@ public async Task<IActionResult> Create(SightingsRequest sightingRequest)
         }
     }
 
+    [HttpGet("")]
+    public ActionResult<SightingListResponse> GetApproved()
+    {
+        try
+        {
+            List<Sighting> sightings = _sightingService.GetApproved();
+            SightingListResponse sightingListResponse = new SightingListResponse();
+            sightingListResponse.SetList(sightings);
```

Comment:
I think all this logic should actually be happening in a method in the service that the controller can then call